burn-import/SUPPORTED-ONNX-OPS.md

```diff
@@ -134,7 +134,7 @@ represent the corresponding Burn Op.
 | [RandomUniform][128]             |       ❌       |      ✅      |
 | [RandomUniformLike][129]         |       ❌       |      ✅      |
 | [Range][130]                     |       ❌       |      ✅      |
-| [Reciprocal][131]                |       ❌       |      ❌      |
+| [Reciprocal][131]                |       ✅       |      ✅      |
```

Comment:
Sorry I should have been clearer. "Import Support" column for Reciprocal should remain ❌ since ONNX op is not yet implemented but Burn Support will have ✅.

Reply:
Hmm, what else do we need for Import Support, if the changes in burn-import/src/burn/node/unary.rs and burn-import/src/onnx/to_burn.rs are not enough? I've tried to follow a couple of implemented op, but I couldn't find anything else?

Reply:
No, you do not need to implement onnx import. We can take this up in a next PR. I just want to indicate now it would be supported by Burn (core) API. The scope of the change is only the documentation.

Reply:
@gzsombor I am sorry I didn't realize you had implemented burn-import for the recip operator. I missed your earlier commits related import crate. So you fully implemented the ONNX op but a test was missing from onnx-tests sub-crate. I went ahead and added it (if you don't mind). I just copied an existing test for `tanh` and modified it.

Reply:
Thank you for your help! 
Yes, I wasn't sure, that it's enough for the burn-import, as I've just made a 'smart-copy and faster' looked after similar operations, and wasn't able to figure out the python stuff - and how to test it.
After your changes, the build is green now! :smile: